mithril-aggregator/src/certifier_service.rs

```diff
@@ -58,6 +58,14 @@ pub enum CertifierServiceError {
     /// Codec error.
     #[error("codec error: '{0}'")]
     Codec(String),
+
+    /// No certificate for this epoch
+    #[error("There is an epoch gap between the last certificate epoch and now ({0:?})")]
+    CertificateEpochGap(Epoch),
```

Comment:
Which epoch is it exactly? (last certificate, current or gap). Maybe you can add a structure with all of them :slightly_smiling_face: